File: Programmers/네트워크.py
# 네트워크

def dfs(computers, check, v):
    check[v] = 1
    for i in range(len(check)):
        if check[i] == 0 and computers[v][i] == 1:
            dfs(computers, check, i)
        
def solution(n, computers):
    answer = 0
    
    check = [0] * n
    
    for i in range(n):
        if check[i] == 0:
            dfs(computers, check, i)
            answer += 1
            
    return answer

File: Programmers/DFS, BFS/네트워크.js
https://programmers.co.kr/learn/courses/30/lessons/43162?language=javascript

function solution(n, computers) {
    var answer = 0;

    let check = Array.from({ length: n }, () => 0);

    for (let i = 0; i < n; i++) {
        if (check[i] == 0) {
            dfs(i, computers, check);
            answer += 1;
        }
    }

    return answer;
}

function dfs(v, computers, check) {
    check[v] = 1;
    for (let i = 0; i < check.length; i++) {
        if (check[i] == 0 & computers[v][i] == 1) {
            dfs(i, computers, check);

        }
    }
}